### ISSUE TITLE
Usage of attention masks

### ISSUE THREAD

**Issue by MahmoudAshraf97** [futo-org/whisper-acft#11]
Hello and thanks for your work, I was wondering if we could use attention masks to enable batching in the training process as it's currently uses a batch size of 1 which is very inefficient, do you think it will give the same result given that whisper was not trained initially using attention masks?
